app/graphql/client.tsx

```diff
@@ -124,6 +130,16 @@ const GaloyClient: React.FC<PropsWithChildren> = ({ children }) => {
         },
       })
 
+      const retry401ErrorLink = new RetryLink({
+        attempts: {
+          max: 3,
+          retryIf: (error) => {
+            return error && error.statusCode === 401
+          },
+        },
+        delay: (count) => count * 20000,
```

Comment:
intent is to wait for 1 min? (0 * 20 + 1 * 20 + 2 * 20 ?)

Reply:
Intent is to retry at 20 seconds, 40 seconds, and 1 minute. I actually think the expression should be delay `delay: 20000`